src/main/java/io/anserini/analysis/vectors/lexlsh/LexicalLSHAnalyzer.java

```diff
@@ -0,0 +1,66 @@
+package io.anserini.analysis.vectors.lexlsh;
```

Comment:
`LexicalLSHAnalyzer` or `LexicalLshAnalyzer`? Strict Java conventions dictate the latter, but both look equally ugly to me...

Reply:
ok for me to adjust name to the latter.